### PR TITLE
Fixing problems with the Ignored Events from Documents

### PR DESCRIPTION
**Pull request by bordoni**
_See the Findings Doc._

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | Expected 1 space after &quot;=&quot;; 2 found | [Ignored_Events.php:96](https://github.com/moderntribe/the-events-calendar/blob/26fbdcc4e846449828a8ec7abbd56f0a892ca403/src/Tribe/Ignored_Events.php#L96)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Ignored_Events.php [hunk 1]
```diff
@@ -50,12 +51,113 @@ private function __construct() {
...
+			// Modify Success messages
+			add_filter( 'bulk_post_updated_messages', array( $this, 'filter_updated_messages' ), 10, 2 );
+
 			/**
 			 * Register Notices
 			 */
 			tribe_notice( 'legacy-ignored-events', array( $this, 'render_notice_legacy' ), 'dismiss=1&type=warning' );
 		}
 
+		public function action_restore_ignored() {
```
Comment (borkweb) [inline src/Tribe/Ignored_Events.php]: A docblock for this method would be well received.

**Comment by borkweb** [comment]
:+1:

File: src/Tribe/Ignored_Events.php (hunk 1 above)
Comment (bordoni) [inline src/Tribe/Ignored_Events.php]: This whole File needs more Docblocks, I will do it later on this week. Sounds good?

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $screen is never used | [Ignored_Events.php:160](https://github.com/moderntribe/the-events-calendar/blob/e9c5f893985dc5e0b5248960c23092d95c523145/src/Tribe/Ignored_Events.php#L160)
:x: | Visibility must be declared on property &quot;$sendback&quot; | [Ignored_Events.php:189](https://github.com/moderntribe/the-events-calendar/blob/e9c5f893985dc5e0b5248960c23092d95c523145/src/Tribe/Ignored_Events.php#L189)
:x: | There must not be more than one property declared per statement | [Ignored_Events.php:192](https://github.com/moderntribe/the-events-calendar/blob/e9c5f893985dc5e0b5248960c23092d95c523145/src/Tribe/Ignored_Events.php#L192)
:x: | Visibility must be declared on property &quot;$sendback&quot; | [Ignored_Events.php:192](https://github.com/moderntribe/the-events-calendar/blob/e9c5f893985dc5e0b5248960c23092d95c523145/src/Tribe/Ignored_Events.php#L192)
:x: | You must have a space after a closing parenthesis | [Ignored_Events.php:244](https://github.com/moderntribe/the-events-calendar/blob/e9c5f893985dc5e0b5248960c23092d95c523145/src/Tribe/Ignored_Events.php#L244)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [Ignored_Events.php:245](https://github.com/moderntribe/the-events-calendar/blob/e9c5f893985dc5e0b5248960c23092d95c523145/src/Tribe/Ignored_Events.php#L245)
:x: | You must have a space after a closing parenthesis | [Ignored_Events.php:250](https://github.com/moderntribe/the-events-calendar/blob/e9c5f893985dc5e0b5248960c23092d95c523145/src/Tribe/Ignored_Events.php#L250)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [Ignored_Events.php:251](https://github.com/moderntribe/the-events-calendar/blob/e9c5f893985dc5e0b5248960c23092d95c523145/src/Tribe/Ignored_Events.php#L251)
:warning: | The method parameter $check is never used | [Ignored_Events.php:629](https://github.com/moderntribe/the-events-calendar/blob/e9c5f893985dc5e0b5248960c23092d95c523145/src/Tribe/Ignored_Events.php#L629)

> via jenkins/codesniffer-phpcs

**Comment by borkweb** [comment]
:huzzah!: :fake-emojis:

**Pull request merged**